src/rebar_hex_repos.erl

```diff
@@ -68,33 +69,37 @@ repos(HexConfig) ->
             merge_repos(RepoList ++ [HexDefaultConfig])
     end.
 
-%% merge repos must add a field repo_name to work with hex_core 0.4.0
+%% merge repos must add a field repo_name to work with r3_hex_core 0.4.0
```

Comment:
comment outdated, I think we're on 0.5.0 now

Reply:
Got it, changed to 0.5.0